### PR TITLE
Fix project not being configurable

### PR DESCRIPTION
**Pull request by oliver-stoehr**

### CODE REVIEW

File: Kitodo/src/test/java/org/kitodo/selenium/testframework/helper/WebDriverProvider.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     // https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
     // Please don't rely on the LATEST_RELEASE file without a version suffix.
     // It exists for backward compatibility only, and will be removed in the near future.
-    private static final String CHROME_DRIVER_LATEST_RELEASE_URL = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_83";
+    private static final String CHROME_DRIVER_LATEST_RELEASE_URL = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_85";
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/selenium/testframework/helper/WebDriverProvider.java:38]: Should this change not be moved into a separate pull request which is first merged before other pull requests get merged so that selenium execution is working again?

**Review by matthias-ronge — CHANGES_REQUESTED** [review]
This would bring back bug #3340 (aka. [edit war](https://en.wikipedia.org/wiki/Wikipedia:Edit_warring)). These changes were part of the fix.

**Comment by Kathrin-Huber** [comment]
Actually #3340 is not fixed. I'm going to test this PR
@oliver-stoehr  can you please add a description to this PR what exactly is being fixed?

**Comment by oliver-stoehr** [comment]
This PR fixes the folder use selection menus but does add #3340 again, #3976 should fix both issues.
